.github/workflows/_build_rosetta.yaml

```diff
@@ -0,0 +1,60 @@
+name: ~build Rosetta container
+
+on:
+  workflow_call:
+    inputs:
+      BASE_IMAGE:
+        type: string
+        description: 'Base docker image that provides T5X'
+        required: false
+        default: ghcr.io/nvidia/t5x:latest
+      BUILD_DATE:
+        type: string
+        description: "Build date in YYYY-MM-DD format"
+        required: false
+        default: 'NOT SPECIFIED'
+    outputs:
+      DOCKER_TAGS:
+        description: "Tags of the image built"
+        value: ${{ jobs.build.outputs.DOCKER_TAGS }}
+
+env:
+  UPLD_IMAGE: ghcr.io/nvidia/jax-toolbox-internal
+
+permissions:
+  contents: read  # to fetch code
+  actions:  write # to cancel previous workflows
+  packages: write # to upload container
+
+jobs:
+
+  build:
+    outputs:
+      DOCKER_TAGS: ${{ steps.meta.outputs.tags }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Print environment variables
+        run: env
+
+      - name: Check out the repository under ${GITHUB_WORKSPACE}
+        uses: actions/checkout@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set docker metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ${{ env.UPLD_IMAGE }}
```

Comment:
Since these are all single-line arguments, why not just make them denser by keeping them in-line?

Reply:
done